utils_nlp/language_utils/hi/hindi_stemmer.py

```diff
@@ -0,0 +1,101 @@
+#! /usr/bin/env python3.1
```

Comment:
please cite the source of this file.

Reply:
I kept the original script as it is, which already contains copyright and citation info.

Reply:
I meant to ask if you downloaded the code from https://research.variancia.com/hindi_stemmer/ and if so, link this page.

Reply:
Added the link.